src/nodes.coffee

```diff
@@ -1431,9 +1430,8 @@ exports.Param = class Param extends Base
     return @reference if @reference
     node = @name
     if node.this
-      node = node.properties[0].name
-      if node.value.reserved
-        node = new Literal o.scope.freeVariable node.value
+      name = "at_#{node.properties[0].name.value}"
```

Comment:
Moreover, it breaks the consistency that all generated variables start with a single `_`, and all helper functions start with two `__`.